react-main/src/app/components/Login/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import actionCreators from '../../../redux/login/actions';
+
+import FormInput from './layout';
+
+class Login extends Component {
+  handleSubmit = values => (
+      this.props.login(values)
+  );
+
+  render() {
+    return (
+      <FormInput onSubmit={this.handleSubmit} />
+    );
+  }
+}
+
+const mapStateToProps = state => ({
+  token: state.login.token
+});
+
+const mapDispatchToProps = dispatch => ({
+  login: (credentials) => dispatch(actionCreators.login({email: credentials.email, password: credentials.password}))
```

Comment:
objects should have space at the begining and the end:
`{ foo: 'foo', bar: 'bar' }` instead of {foo:'foo', bar:'bar'}